### PR TITLE
@broskoski: Some artwork page attributes

### PR DESCRIPTION
**Pull request by dzucconi**

### CODE REVIEW

File: schema/artwork/index.js [hunk 1]
```diff
@@ -44,13 +48,28 @@ const ArtworkType = new GraphQLObjectType({
...
-      can_share_image: {
-        type: GraphQLBoolean,
-      },
       partner: {
         type: Partner.type,
         resolve: ({ partner }) => gravity(`partner/${partner.id}`),
       },
+      can_share_image: {
+        type: GraphQLBoolean,
+        deprecationReason: 'Favor `is_`-prefixed boolean attributes',
```
Comment (broskoski) [inline schema/artwork/index.js]: Interesting, is this used anywhere now?

File: schema/artwork/utilities.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    isDimensional(height) &&
+    !isThreeDimensional({ dimensions })
+  );
+};
+
+export const isTooBig = ({ dimensions }) => {
+  const LIMIT_IN_CM = 1500;
+  const { width, height } = mapValues(dimensions.cm, parseFloat);
+  return width > LIMIT_IN_CM || height > LIMIT_IN_CM;
+};
```
Comment (broskoski) [inline schema/artwork/utilities.js]: Into all these

File: schema/artwork/index.js (hunk 1 above)
Comment (dzucconi) [inline schema/artwork/index.js]: Looks like the show2 page. By deprecating it still works but no longer shows up in GraphiQL

**Comment by broskoski** [comment]
Very nice :+1:

**Pull request merged**